inference-engine/src/plugin_api/cpp_interfaces/impl/ie_executable_network_internal.hpp

```diff
@@ -73,8 +73,6 @@ class ExecutableNetworkInternal : public IExecutableNetworkInternal {
 
     void Export(std::ostream& networkModel) override {
         std::stringstream strm;
-        strm.write(exportMagic.data(), exportMagic.size());
-        strm << _plugin->GetName() << std::endl;
         ExportImpl(strm);
         networkModel << strm.rdbuf();
```

Comment:
looks like since we removed magic with device name header, we can remove `ExportImpl` at all and implement `Export` directly in plugins. Otherwise, we write to streams twice (temporary `stringstream` and final `networkModel` stream).

Can be done as a separate task.